src/operations/groupBy.ts

```diff
@@ -48,12 +51,27 @@ export function groupValuesByColumn({
   const targetOperation =
     aggregationFunctions[operation as Exclude<AggregationType, 'custom'>];
 
-  if (targetOperation) {
-    return Array.from(groups).map(([name, value]) => ({
-      name,
-      value: targetOperation(value),
-    }));
+  if (!targetOperation) {
+    return [];
   }
 
-  return [];
+  const allCategories = Array.from(groups).map(([name, value]) => ({
+    name,
+    value: targetOperation(value),
+  }));
+
+  allCategories.sort((a, b) => b.value - a.value);
+
+  if (othersThreshold && allCategories.length > othersThreshold) {
+    const otherNames = allCategories
+      .map((entry) => entry.name)
+      .slice(othersThreshold);
+    const otherValue = otherNames.flatMap((name) => groups.get(name));
```

Comment:
As `othersThreshold` is usually < 20, there shouldn't be any performance impact, but you're iterating twice when computing `otherNames` and `otherValue`, so feel free to do something like:

```
const others = allCategories.slice(othersThreshold);
const otherValues = others.flatMap(({ name }) => groups.get(name));
```